src/Countdown.elm

```diff
@@ -0,0 +1,103 @@
+module Main exposing (Model, Msg(..), init, main, subscriptions, update, view)
+
+import Browser
+import Html
+import Task
+import Time
+
+
+main : Program () Model Msg
+main =
+    Browser.element
+        { init = init
+        , update = update
+        , view = view
+        , subscriptions = subscriptions
+        }
+
+
+type Msg
+    = Tick Time.Posix
+    | AdjustTimeZone Time.Zone
+    | CurrentTime Time.Posix
+
+
+type alias Model =
+    { zone : Time.Zone
+    , time : Time.Posix
+    }
+
+
+init : flags -> ( Model, Cmd Msg )
+init _ =
```

Comment:
Instead of consuming an unnecessary argument, use `always`:
```elm
init : (Model, Cmd Msg)
init = ...
```
and in `main`:
```elm
    Browser.element
        { init = always init
        ...
```